x-pack/docs/en/ilm/start-stop-ilm.asciidoc

```diff
@@ -0,0 +1,166 @@
+[[start-stop-ilm]]
+== Start And Stop {ilm}
+
+All indices that are managed by ILM will continue to run through
```

Comment:
nit: Can we say "execute" rather than "run through"?

Reply:
yup!